Tests/StencilSwiftKitTests/ParseBoolTests.swift

```diff
@@ -4,21 +4,24 @@
 // MIT Licence
 //
 
-import XCTest
 @testable import StencilSwiftKit
+import XCTest
 
 class ParseBoolTests: XCTestCase {
-
   func testParseBool_TrueWithString() throws {
+    // swiftlint:disable force_unwrapping
     XCTAssertTrue(try Filters.parseBool(from: ["true"])!)
```

Comment:
ditto https://github.com/SwiftGen/StencilSwiftKit/pull/78#discussion_r177750011

Reply:
Same as in https://github.com/SwiftGen/SwiftGen/pull/402: these changes will disappear once #78 is merged. Also applies to the comments below